pgx/src/io_intf.ml

```diff
@@ -5,7 +5,9 @@ module type S = sig
   val (>>=) : 'a t -> ('a -> 'b t) -> 'b t
   val catch : (unit -> 'a t) -> (exn -> 'a t) -> 'a t
   type in_channel
+  val sexp_of_in_channel : in_channel -> Sexplib0.Sexp.t
   type out_channel
+  val sexp_of_out_channel : out_channel -> Sexplib0.Sexp.t
```

Comment:
Why do we need these to be sexp-able? Couldn't we just declare them as `sexp_opaque`?